spikeinterface/extractors/utils/mdaio.py

```diff
@@ -0,0 +1,544 @@
+import numpy as np
```

Comment:
I would copy/paste all that function inside the mdaextractors.
Even if it make a super file. So evrything is centralise and more easy to manage.
No other module will depend on this mdaio.
Am I wring ?